app/views/user_mailer/in_person_ready_to_verify.html.erb

```diff
@@ -0,0 +1,100 @@
+<p>
+  <%= t('user_mailer.in_person_ready_to_verify.greeting', name: @first_name) %><br>
+  <%= t('user_mailer.in_person_ready_to_verify.intro') %>
+</p>
+
+<table class="info-alert margin-y-4">
```

Comment:
Is it possible to make the alert heading: 
- the same font size as the alert body copy
- bolded

I think this is one way to introduce some consistency in style between the email and the Login page.

Reply:
The font size difference is present in the Figma designs (18px for bolded heading, 16px for unbolded text). It'd be easy enough for me to standardize them as the same body font size though if that's what we want.

Reply:
I can see how it's a bit more noticeable a difference to control these font sizes in the email template though, since the default body font size is 13px in the emails (vs. 16px elsewhere in the app).